html/modules/custom/reliefweb_import/src/Command/ReliefwebImportCommand.php

```diff
@@ -0,0 +1,581 @@
+      WHERE bundle = 'job'
+      AND field_job_id_value = :id";
+
+    $count = $this->database->query($sql, [
+      ':id' => $id,
+    ])->fetchCol();
+
+    return $count[0] != 0;
+  }
+
+  /**
+   * Load job by Id.
+   */
+  protected function loadJobById($id) {
+    $sql = "SELECT entity_id
+      FROM {node__field_job_id}
+      WHERE bundle = 'job'
+      AND field_job_id_value = :id";
+
+    $nid = $this->database->query($sql, [
+      ':id' => $id,
+    ])->fetchCol();
+
+    return Job::load(reset($nid));
+  }
+
+  /**
+   * Create a new job.
+   */
+  protected function createJob($data) {
+    $values = [
+      'type' => 'job',
+      'uid' => 1,
+      'field_job_id' => (string) $data->link,
+      'title' => (string) $data->title,
+      'field_career_categories' => $data->field_career_categories[0] ? (array) $data->field_career_categories : 0,
+      'field_city' => (string) $data->field_city,
+      'field_job_closing_date' => (string) $data->field_job_closing_date,
+      'field_country' => $this->mapCountries((array) $data->field_country),
+      'field_how_to_apply' => (string) $data->field_how_to_apply,
+      'body' => [
+        'value' => $this->validateBody((string) $data->body),
+        'format' => 'markdown',
+      ],
+      'field_job_type' => $data->field_job_type[0] ? (array) $data->field_job_type : 0,
+      'field_job_experience' => $data->field_job_experience[0] ? (array) $data->field_job_experience : 0,
+      'field_source' => $data->field_source[0] ? (array) $data->field_source : 0,
+      'field_theme' => $data->field_theme[0] ? (array) $data->field_theme : 0,
+    ];
+
+    $job = Job::create($values);
+
+    $this->validateAndSaveJob($job);
+  }
+
+  /**
+   * Create a new job.
+   */
+  protected function updateJob(Job $job, $data) {
+    $job->title = (string) $data->title;
+    $job->field_career_categories = $data->field_career_categories[0] ? (array) $data->field_career_categories : 0;
+    $job->field_city = (string) $data->field_city;
+    $job->field_job_closing_date = (string) $data->field_job_closing_date;
+    $job->field_country = $this->mapCountries((array) $data->field_country);
+    $job->field_how_to_apply = (string) $data->field_how_to_apply;
+    $job->body = [
+      'value' => $this->validateBody((string) $data->body),
+      'format' => 'markdown',
+    ];
+    $job->field_job_type = $data->field_job_type[0] ? (array) $data->field_job_type : 0;
+    $job->field_job_experience = $data->field_job_experience[0] ? (array) $data->field_job_experience : 0;
+    $job->field_source = $data->field_source[0] ? (array) $data->field_source : 0;
+    $job->field_theme = $data->field_theme[0] ? (array) $data->field_theme : 0;
+
+    $this->validateAndSaveJob($job);
+  }
+
+  /**
+   * Validate and save job.
+   */
+  protected function validateAndSaveJob(Job $job) {
+    $violations = $job->validate();
+    if (count($violations) === 0) {
+      $job->setNewRevision(TRUE);
```

Comment:
The revision user should be set to `2` (the system user): `$job->setRevisionUserId(2)`.